backends/pkg/reconciler/eventmesh/handler.go

```diff
@@ -23,15 +31,22 @@ type EventMesh struct {
 	Brokers    []*Broker    `json:"brokers"`
 }
 
-type EventTypeMap = map[string]*EventType
+type Subscription struct {
+	BackstageIds map[string]struct{}
+}
+
+// SubscriptionMap key: "<namespace>/<eventType.spec.type>"
+type SubscriptionMap map[string]Subscription
```

Comment:
Maybe a more specific name like `SubscriptionByNamespacedType` ?

Reply:
I removed this map when I simplified the trigger processing